deployment/environments/dev/variables.tf

```diff
@@ -34,6 +34,12 @@ variable "notifications_email" {
     default = "no-reply@aspect.dev"
 }
 
+variable "debug_email" {
+    type = string
+    description = "Email address to notify development team of issues"
+    default = "derek@aspect.dev"
```

Comment:
nit: maybe we should create a Google Group so you're not the only one on the distribution list?

Reply:
Yeah that would be nice. Can you set up `publish-to-bcr-dev@aspect.com` or something of the sort?

Reply:
Oh, you meant a Google group, not a mailing group? Though maybe those aren't so different. I'll look into it.